src/data_api/orders.se

```diff
@@ -51,6 +54,8 @@ def checkHash(orderHash, sender):
 
 def saveOrder(orderID, type, market, fxpAmount, fxpPrice, sender, outcome, fxpMoneyEscrowed, fxpSharesEscrowed, tradeGroupID):
     self.controller.assertIsWhitelisted(msg.sender)
+    if(outcome > MARKETS.getMarketNumOutcomes(market)):
+        ~invalid()
```

Comment:
@joeykrug This was something you added to the reporting refactor branch.  It didn't seem related to the reporting stuff so I put it here, but I can move it over to that PR instead if that makes more sense.